### PR TITLE
i18n: translated images

### PR DESCRIPTION
**Pull request by nhsz**
This PR updates `rehypeImgSize` plugin to set translated images as src when available

## Description

- updates `rehypeImgSize` assigned src logic
- adds `locale` parameter to `rehypeImgSize`
- adds `checkIfImageIsTranslated` and `getTranslatedImgPath` to `/utils/i18n`
- adds `TRANSLATED_IMAGES_DIR` constant
- adds `/web3` to list of allowed to build pages (and `enables QuizWidget` component on Static layout)

## Example pages that contain some translated images

- [/es/energy-consumption](https://deploy-preview-56--ethereum-org-fork.netlify.app/es/energy-consumption)
- [/es/web3](https://deploy-preview-56--ethereum-org-fork.netlify.app/es/web3)

### CODE REVIEW

File: src/lib/rehype/rehypeImgSize.ts
Comment (pettinarip) [inline src/lib/rehype/rehypeImgSize.ts:1]: This plugin could be renamed to something more generic now.

**Review by pettinarip — APPROVED** [review]
🚀 great!

File: src/lib/rehype/rehypeImgSize.ts
Comment (nhsz) [reply src/lib/rehype/rehypeImgSize.ts:1]: Agree, something like `rehypeImg` could work

Comment (pettinarip) [reply src/lib/rehype/rehypeImgSize.ts:1]: Sounds good!

**Pull request merged**